openworld/sdk/generator/resources/requirements.txt

```diff
@@ -0,0 +1,6 @@
+uri~=2.0.1
+furl~=2.1.3
+openworld-sdk-python-core~=0.3.0
+pydantic~=1.10.4
+pydantic[email]
```

Comment:
Please add the version.

Reply:
As pydantic version is already set, it will use it for the pydantic email validator (pydantic[email]).